docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -183,7 +278,7 @@ Some proxies require authentication. The format of the `HTTP_PROXY` or `HTTPS_PR
 
 If you run Azure CLI on a build machine where multiple jobs can be run in parallel, access tokens might be shared between two build jobs run as the same OS user.  To avoid mix ups, set `AZURE_CONFIG_DIR` to a directory where the access tokens are stored.
 
-## Generic update arguments
+## Generic update parameters
```

Comment:
In-tool helps uses the word `argument`. Any reason for the change?

```
> az group create -h

Command
    az group create : Create a new resource group.

Arguments
    --location -l                 [Required] : Location. Values from: `az account list-locations`.
                                               You can configure the default location using `az
                                               configure --defaults location=<location>`.
    --name --resource-group -g -n [Required] : Name of the new resource group.
    --managed-by                             : The ID of the resource that manages this resource
                                               group.
    --tags                                   : Space-separated tags: key[=value] [key[=value] ...].
                                               Use "" to clear existing tags.
```

Reply:
@jiashli,  I don't know why the docs use `parameter` (a placeholder for a value) when the in-tool help uses `arguments` (the value passed), but conceptual and autogen docs should use the same word.  Even `az config param-persist` kept the word `parameter`.

Technicall, the `--location` parameter can contain the following `arguments`: eastus, westus